components/src/atoms/button/Button.stories.js

```diff
@@ -1,94 +1,60 @@
 import { storiesOf } from '@storybook/vue'
+import { select, text } from '@storybook/addon-knobs'
 
 import AButton from './Button.vue'
 import AIcon from '../icon/Icon.vue'
 
 const info = `
   ---
-  #### Use these modifiers with \`.button\` class.
+  ###### Check **Knobs** tab to edit component properties dynamically. Below list of available BEM modifiers.
   - \`.button--secondary\` - Selector for applying secondary styles
   - \`.button--blank\` - Selector for applying blank styles
   - \`.button--fluid\` - Selector for applying fluid styles (width: 100%)
   ---
 `
 
+const bemModifiers = [
+  'null',
+  'button--secondary',
+  'button--blank',
+  'button--fluid'
+]
+
 storiesOf('Atoms/Button', module)
   .addParameters({ info })
-  // @vue/component
```

Comment:
Afaik we need this comment for eslint

Reply:
It doesn't make sense to lint this part of code. It show error when I don't set props types.